fre/yamltools/tests/test_combine_yamls.py

```diff
@@ -0,0 +1,59 @@
+import os
+from pathlib import Path
+import pytest
+from fre.yamltools import combine_yamls as cy
+
+## SET-UP
+# Set example yaml paths, input directory
+CWD = Path.cwd()
+yamls_dir = Path("fre/yamltools/tests/AM5_example")
+in_dir = Path(f"{CWD}/{yamls_dir}")
+
+# Set what would be click options
+EXPERIMENT = "c96L65_am5f7b12r1_amip"
+PLATFORM = "gfdl.ncrc5-intel22-classic"
+TARGET = "prod"
+
+def test_modelyaml_exists():
+    """
+    Make sure main yaml file exists
+    """
+    assert Path(f"{CWD}/{yamls_dir}/am5.yaml").exists()
+
+def test_compileyaml_exists():
+    """
+    Make sure experiment yaml file exists
+    """
+    assert Path(f"{CWD}/{yamls_dir}/compile.yaml").exists()
+
+def test_platformyaml_exists():
+    """
+    Make sure experiment yaml file exists
+    """
+    assert Path(f"{CWD}/{yamls_dir}/yaml_include/platforms.yaml").exists()
+
+def test_expyaml_exists():
+    """
+    Make sure experiment yaml file exists
+    """
+    assert Path(f"{CWD}/{yamls_dir}/yaml_include/pp.c96_amip.yaml").exists()
+
+@pytest.mark.skip(reason='analysis scripts might not be defined')
+def test_analysisyaml_exists():
+    """
+    Make sure experiment yaml file exists
+    """
+    assert Path(f"{CWD}/{yamls_dir}/yaml_include/analysis.yaml").exists()
+
+def test_merged_yamls():
+    """
+    Check for the creation of the combined-[experiment] yaml 
+    Check that the model yaml was merged into the combined yaml
+    """
+    os.chdir(in_dir)
+
+    modelyaml = Path(f"{CWD}/{yamls_dir}/am5.yaml")
```

Comment:
looking at the top- can we remove `os.chdir` here in favor of making the later line, equivalently, `modelyaml = Path(f"{in_dir}/{yamls_dir}/am5.yaml")`?

If we can't b.c. this makes the test fail, i'd call that a fragility that is worth exploring. Either as part of this PR, or opening a sep issue as needed

Reply:
wow- it did break the test didn't it? not fun

I'm now worried about this too: https://github.com/NOAA-GFDL/fre-cli/blob/main/fre/pp/tests/test_configure_script_yaml.py#L30-L53

Reply:
If this works fine with the `os.chdir` there, we can merge it in. I can open a separate issue for this odd dependence on `os.chdir` in different `fre yaml` routines

Reply:
lol, yeah although I thought maybe it was related to some path in the action. Like I thought I've seen this before but I basically made this test similar to the configure_script_yaml one. So now I'm trying to troubleshoot what happened and how things broke.

Reply:
If I can't figure it out today, I can add that `os.chdir` back in